mojo-data/mojos/rulefit/cars/v1.00/rulefit_v1.00_cars_scores0.txt

```diff
@@ -0,0 +1,406 @@
+[16.481831750233788, 0.0]
+[14.495768119092752, 0.0]
```

Comment:
zeros here are expected because pred size is 2 for GenModel because 
```
  @Override public int getPredsSize() { // fractional binomial has numerical response
    return isClassifier() ? (1 + getNumResponseClasses()) : 2;
  }
```
applies for all regression cases.